egs/timit/ssr1/featprepare.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
```

Comment:
Can you move this script to `local/` and modify `run.sh` accordingly?

Reply:
Got it! Thanks for your response!